DQMOffline/Trigger/plugins/HTMonitor.cc

```diff
@@ -195,28 +211,74 @@ void HTMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup)
   }
   if ( muons.size() < nmuons_ ) return;
 
-  // filling histograms (denominator)  
-  htME_variableBinning_.denominator -> Fill(ht);
-  deltaphimetj1ME_.denominator -> Fill(deltaPhi_met_j1);
-  deltaphij1j2ME_.denominator -> Fill(deltaPhi_j1_j2);
+  // fill histograms
+  if(quantity_ == "HT")
```

Comment:
please make a set of enums for quantity_ so that the string comparison is done once at the start of the job instead of in each event.